mysql/const.go

```diff
@@ -422,6 +441,9 @@ var SetStr2Priv = map[string]PrivilegeType{
 	"Show View":   ShowViewPriv,
 }
 
+// AllGlobalPrivs is all the privileges in global scope.
+var AllGlobalPrivs = []PrivilegeType{SelectPriv, InsertPriv, UpdatePriv, DeletePriv, CreatePriv, DropPriv, ProcessPriv, GrantPriv, ReferencesPriv, AlterPriv, ShowDBPriv, SuperPriv, ExecutePriv, IndexPriv, CreateUserPriv, TriggerPriv, CreateViewPriv, ShowViewPriv, CreateRolePriv, DropRolePriv, CreateTMPTablePriv, LockTablesPriv, CreateRoutinePriv, AlterRoutinePriv, EventPriv}
```

Comment:
`GrantPriv` has been removed from `AllGlobalPrivs`, see #665 for detail.

Reply:
done